code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -272,28 +272,44 @@ var/global/chicken_count = 0
 	else
 		SSobj.processing.Remove(src)
 
-/obj/udder
 
-/obj/udder/New()
+/obj/item/udder

```

Comment:
Would this be better as a /obj/item/organ/internal or do you guys think that'd be overkill?